### PR TITLE
 Fix broken link to page of historical relevance (2)

### LINKED ISSUE #126: Nighlty linkcheck failed unexpectedly on Web Archive URL
## Expected behavior

The nightly run of `make docs-linkcheck` succeeds unless a URL is broken. (The job was introduced in #119.)

## Actual behavior

It failed, see [**linkcheck #170**][error].

## Steps to reproduce the problem

- Re-building the nightly job may reproduce the error ([**linkcheck #170**][error])

## Additional information

The cause of [the error][error] is not obvious to me:

```
Warning, treated as error:
/root/project/docs/getting_the_most_out_of_securedrop.rst.rst:86:broken link: https://web.archive.org/web/20201114212453/https://specialprojectsdesk.com/tell-on-trump-1792401813 (404 Client Error: Not Found for url: https://web.archive.org/web/20201118014447/https://specialprojectsdesk.com/tell-on-trump-1792401813)
make: *** [Makefile:17: docs-linkcheck] Error 2
```

Note the dates: `20201114212453` in the URL [that's the URL in the docs][link], and `20201118014447` in the URL that reports the `404 Client Error`.

I can't reproduce the error from a browser, I get a consistent HTTP 200 for the former URL. (The latter does indeed respond with HTTP 404, but I don't know yet how 'linkcheck' got there, I see no redirects in my browser logs.)

  [error]: https://app.circleci.com/pipelines/github/freedomofpress/securedrop-docs/174/workflows/218c1ca4-63a8-459a-9cfa-a4f3f021b43c/jobs/170
  [link]: https://github.com/freedomofpress/securedrop-docs/pull/118/files#diff-36ecf56ad6fe94f1f43551c7090ba8d17dbaf0faa64c28d3f96b8525a388478bR94

### PR DESCRIPTION
**Pull request by gonzalo-bulnes**
## Status

Ready for review

## Description of Changes

This PR is a follow up on #118. I misspelled the timestamp in the link :disappointed: 

Fixes #126 

## Testing

- [ ] This time, when running `make docs-linkcheck` the logs should report `ok` instead of `redirect / found`.
```
(line   86) ok        https://web.archive.org/web/20200926063152/https://specialprojectsdesk.com/tell-on-trump-1792401813
```

## Release 

I can't think of any special consideration before releasing this change.

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [x] Doc link linting (`make docs-linkcheck`) passed  (not only passed but doesn't report the tell-tale redirect)
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/getting_the_most_out_of_securedrop.rst [hunk 1]
```diff
@@ -91,7 +91,7 @@ proof of concept Twitter advertisement aimed at EPA and NOAA employees to show
 how it can be done. You can read about `how you can do the same thing here`_.
 
 .. _`recently targeted online advertisements`: https://www.wsj.com/articles/gizmodo-ads-target-potential-trump-leakers-1487191482
-.. _`tell on trump`: https://web.archive.org/web/20201114212453/https://specialprojectsdesk.com/tell-on-trump-1792401813
+.. _`tell on trump`: https://web.archive.org/web/20201114214553/https://specialprojectsdesk.com/tell-on-trump-1792401813
```
Comment (zenmonkeykstop) [inline docs/getting_the_most_out_of_securedrop.rst:94]: This returns a 200 from the IA, but the recorded page is a Kinja 404. Using an earlier version like https://web.archive.org/web/20200926063152/https://specialprojectsdesk.com/tell-on-trump-1792401813 works.

**Review by zenmonkeykstop — CHANGES_REQUESTED** [review]

File: docs/getting_the_most_out_of_securedrop.rst (hunk 1 above)
Comment (gonzalo-bulnes) [reply docs/getting_the_most_out_of_securedrop.rst:94]: I picked the last date that didn't display the Kinja 404 page, so the green/corrected link shouldn't  do that (Nov. 14) - though the [misspelled URL sometimes did](https://github.com/freedomofpress/securedrop-docs/issues/126). **That being said**, the September 26 version is just as fine. I'll make the change :+1:

**Comment by gonzalo-bulnes** [comment]
(rebased)

**Review by eloquence — APPROVED** [review]
LGTM and `make docs-linkcheck` succeeds now, thanks for the Internet Archive trial & error.

**Pull request merged**